app/addon_cmds/commands.py

```diff
@@ -139,12 +139,13 @@ def addon(
     """
     Command line to Install/Uninstall Addons on OCM managed cluster.
     """
+    _rosa = [addon_name.strip() for addon_name in rosa.split(",")] if rosa else []
```

Comment:
this is not needed, just do `_rosa = rosa.split(",")`

Reply:
Done, but note that it's space sensitive, not intuitive for all users.
example: 
`--rosa "managed-odh, managed-api-service"` **will raise and abort.**
`--rosa managed-odh,managed-api-service` will pass with/without `""`
@myakove @rnetser Please resolve if it's OK.